alpine/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM alpine:3.4
+
+MAINTAINER Datadog <package@datadoghq.com>
+
+ENV DD_HOME=/opt/datadog-agent \
+    DOCKER_DD_AGENT=yes \
+    AGENT_VERSION=5.8.0 \
+    # prevent the agent from being started after install
+    DD_START_AGENT=0
+
+# Add Docker check
+COPY conf.d/docker_daemon.yaml "$DD_HOME/agent/conf.d/docker_daemon.yaml"
+
+# Add install and config files
+ADD https://raw.githubusercontent.com/DataDog/dd-agent/5.8.0/packaging/datadog-agent/source/setup_agent.sh /setup_agent.sh
+COPY entrypoint.sh /entrypoint.sh
+
+# Expose supervisor and DogStatsD port
+EXPOSE 9001/tcp 8125/udp
+
+# Install minimal dependencies
+RUN apk add -qU --no-cache curl-dev python-dev tar sysstat
+
+# Install build dependencies
+RUN apk add -qU --no-cache -t .build-deps curl gcc musl-dev pgcluster-dev linux-headers \
+    # Install the agent
+    && sh /setup_agent.sh \
+    # Clean build dependencies
+    && apk del -q .build-deps
```

Comment:
1. won't hurt to remove the `setup_agent.sh` file after being done with it.
 1. optional: place "temporary" files like this in `/tmp` instead of root `/`